OpenTelemetry: Fix indentation in collapser for JVMs (#11594)
newrelic/docs-website: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-jvms-page.mdx

    id="jvms-and-metric-types"
    title="Gauges versus counters"
  >
We recommend you use the OpenTelemetry Java agent 1.13.0 or higher, which collects JVM memory usage as an async gauge. These agent versions allow you to view metrics in New Relic without the workarounds you'd need to do if you were using older agents.

> If you're using older OpenTelemetry Java agents (versions 1.10.0 thorugh 1.12.0), keep in mind that JVM memory usage switched from being collected as an async gauge to an async up down counter. This has implications on the exported data. Gauges and counters export differently:

Reviewer: ```suggestion
If you're using older OpenTelemetry Java agents (versions 1.10.0 through 1.12.0), keep in mind that JVM memory usage switched from being collected as an async gauge to an async up down counter. This has implications on the exported data. Gauges and counters export differently:
```

* Async gauges export as OTLP gauges.
* Async up down counters export as OTLP non-monotonic sums.

> If you configure your SDK to export your metrics using delta aggregation temporality (which is required for counter and histogram instruments to function with New Relic), that results in async up down counters exported as non-monotonic delta sums. New Relic can't perform any useful analysis of non-monotonic delta sum data.

Reviewer: ```suggestion
If you configure your SDK to export metrics using delta aggregation temporality (which is required for counter and histogram instruments to function with New Relic), that results in async up down counters exported as non-monotonic delta sums. New Relic can't perform any useful analysis of non-monotonic delta sum data.
```

What is "async up down counters"?

Author: @a-sassman I had thought these were used as "async up down counters" in the OpenTelemetry spec, but alas, it looks like these are "Asynchronous UpDownCounters."

> If you have to use OpenTelemetry Java agent versions 1.10.0 through 1.12.0, the workaround is to use the View API to indicate that async up down counters should be aggregated using [last value aggregation](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#last-value-aggregation) instead of the default sum aggregation. This results in JVM memory usage being exported as gauge data, which is required for a useful experience in New Relic.

Reviewer: Same question - what is "async up down counters"?

Author: @a-sassman see my comment above.

The way you configure the View API varies based on whether you’re using the OpenTelemetry Java agent:

> * If you're not using the OpenTelemetry Java agent, review this simple [example](https://github.com/newrelic/newrelic-opentelemetry-examples/pull/89/files#diff-da355ef6d1092534a55829e95160ab8468884bdd521f9018feeaaa66aea6ac5bR82-R86) that shows how to register a view when configuring `SdkMeterProvider`.

Reviewer: ```suggestion
```

> * If you’re using the OpenTelemetry Java agent, you need to configure the View API in an [extension](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/agent-config.md#extensions). Extensions allow you to hook into the SDK configuration (among other things) and apply programmatic configuration that isn’t available by environment variable or system properties. This [example](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/java/agent-nr-config) demonstrates how you can use an extension to [customize](https://github.com/newrelic/newrelic-opentelemetry-examples/blob/main/java/agent-nr-config/config-extension/src/main/java/com/newrelic/otel/extension/Customizer.java#L28-L37) the `SdkMeterProvider`’s views.

Reviewer: ```suggestion
* If you are using the OpenTelemetry Java agent, you need to configure the View API in an [extension](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/agent-config.md#extensions). Extensions allow you to hook into the SDK configuration (among other things) and apply programmatic configuration that isn’t available by environment variables or system properties. This [example](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/java/agent-nr-config) demonstrates how you can use an extension to [customize](https://github.com/newrelic/newrelic-opentelemetry-examples/blob/main/java/agent-nr-config/config-extension/src/main/java/com/newrelic/otel/extension/Customizer.java#L28-L37) the `SdkMeterProvider`’s views.
* If you're not using the OpenTelemetry Java agent, review this simple [example](https://github.com/newrelic/newrelic-opentelemetry-examples/pull/89/files#diff-da355ef6d1092534a55829e95160ab8468884bdd521f9018feeaaa66aea6ac5bR82-R86) that shows how to register a view when configuring `SdkMeterProvider`.
```

I don't find the link on "[customize](https://github.com/newrelic/newrelic-opentelemetry-examples/blob/main/java/agent-nr-config/config-extension/src/main/java/com/newrelic/otel/extension/Customizer.java#L28-L37" to be applicable to this bullet point. The example in the second sentence seems more relevant. Perhaps consider removing that first example?
